aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -122,48 +122,48 @@ def process_message(record: Dict) -> Dict[str, Union[str, bool]]:
     Ensure the message is proper format. Get the customer
     entitlements and send the information to the SCC service.
     """
+    sqs_event_manager_config = Defaults.get_sqs_event_manager_config()
     result = {
         'itemIdentifier': record.get('messageId') or 'unknown',
         'status': 'unknown',
         'error': True
     }
     try:
         message = AWSSNSMessage(record)
+        auth_token = sqs_event_manager_config.get('auth_token') or ''
```

Comment:
It's a dict as such no `or` required, simply use `sqs_event_manager_config.get('auth_token', '')`

Reply:
done